dist/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+<script defer src="main.js"></script></head>
+<body>
+    <ul class="outter">
+        <li>Today's To Do
+            <i class="fa-solid fa-arrows-rotate"></i>
+        </li>
+        <li>
+            <div class="add-items">
+                <label for="add-item"></label>
+                <input type="text" name="add-item" id="add-item"  placeholder="Add to your list...">
+            </div>
+            <i class="fa-solid fa-arrow-left"></i>
+        </li>
+        <li>
+            <ul class="inner"></ul>
+        </li>
+        
+        <li><button type="submit">Clear all completed</button></li>
+    </ul>
```

Comment:
Try to minimize and simplify this section of your index.html code. Try to use only one <ul> for dynamically adding lists as well as initializing the first two and the last list items.

Reply:
I think it is not that complicated it is designed to be nested as the inner ul is used to be manipulated easily using javascript